### ISSUE TITLE
SecretgenController is referencing a non-existent SHA

### ISSUE THREAD

**Issue by jayunit100** [vmware-tanzu/community-edition#3671]
## Bug Report

problem
          
```image: projects.registry.vmware.com/tce/secretgen-controller@sha256:0e2791ab37768fa1dd843454a1fac9403539930362f128e3a897419fa431d96c```

referenced in:

```https://github.dev/vmware-tanzu/community-edition/blob/9c69a11ccd1ff292666b961dad9a5936afa504cb/addons/packages/secretgen-controller/0.8.0/package.yaml#L15-L16```

doesnt exist !

```
docker pull  projects.registry.vmware.com/tce/secretgen-controller@sha256:0e2791ab37768fa1dd843454a1fac9403539930362f128e3a897419fa431d96c 
```

## Expected Behavior

Any of the images in our addons should be pullable :)

**Comment by seemiller** [comment]
Or maybe the package yaml was never updated...

**Comment by aaronshurley** [comment]
@seemiller Sorry about this. I just looked at [the original PR](https://github.com/vmware-tanzu/community-edition/pull/3563/files) and it looks like the SHA in `package.yaml` is the same as the secretgen-controller image itself. So, I think I mistakenly copied and pasted this one too many times.

I did check my terminal history and saw that I ran a `make push-package PACKAGE=secretgen-controller VERSION=0.8.0 TAG=0.8.0`. So, it's possible that I pushed the package but didn't provide the correct SHA...

Thanks for cleaning up my mess! 😅
